delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/provisioning/ProjectProvisioning.scala

```diff
@@ -63,44 +63,47 @@ object ProjectProvisioning {
     *   provisioning configuration
     */
   def apply(
-      appendAcls: Acl => IO[AclRejection, Unit],
+      appendAcls: Acl => IO[Unit],
       projects: Projects,
       provisioningConfig: AutomaticProvisioningConfig
-  ): ProjectProvisioning = new ProjectProvisioning {
+  )(implicit contextShift: ContextShift[IO]): ProjectProvisioning = new ProjectProvisioning {
 
     private def provisionOnNotFound(
         projectRef: ProjectRef,
         user: User,
         provisioningConfig: AutomaticProvisioningConfig
-    ): IO[ProjectProvisioningRejection, Unit] = {
+    ): IO[Unit] = {
       val acl = Acl(AclAddress.Project(projectRef), user -> provisioningConfig.permissions)
       for {
         _ <- appendAcls(acl)
+               .toBIO[AclRejection]
                .onErrorRecover { case _: AclRejection.IncorrectRev | _: AclRejection.NothingToBeUpdated => () }
                .mapError(UnableToSetAcls)
         _ <- projects
                .create(
                  projectRef,
                  provisioningConfig.fields
-               )(user)
+               )(user, contextShift)
+               .toBIO[ProjectRejection]
```

Comment:
Can all of this be CE now? I guess there's an implicit conversion anyway at the end of the function

Reply:
I left this aside and forgot 😓  Is there a partial recovery/redeem function in CE? Couldn't find that which is why I left it aside in the first place

Reply:
Monix is now gone in this file